index.bs

```diff
@@ -375,6 +375,26 @@ The [=agent certificate=] must have the following characteristics:
     * secp521r1_sha512
 * Valid for signing
 
+Let the <dfn>certificate serial number</dfn> be the result of the following steps:
+
+<ol>
+  <li>If the agent has never generated an agent certificate:
+  <ol>
+    <li>Let the <dfn>certificate serial number base</dfn> be a 32-bit
+        pseudorandom integer value.</il>
+    <li>Let the <dfn>certificate serial number counter</dfn> be a 32-bit
+        unsigned integer.</li>
+    <li>Assign the [=certificate serial number counter=] to 0.</li>
```

Comment:
Nit: the base is declared and assigned in a single step. For consistency, have both the base and counter either declared and assigned in one step each, or two steps.

Reply:
Done